core-bundle/src/EventListener/UnwrapTwigExceptionListener.php

```diff
@@ -12,39 +12,27 @@
 
 namespace Contao\CoreBundle\EventListener;
 
-use Contao\CoreBundle\Exception\NoContentResponseException;
-use Contao\CoreBundle\Exception\RedirectResponseException;
+use Contao\CoreBundle\Exception\ResponseException;
 use Symfony\Component\HttpKernel\Event\ExceptionEvent;
 use Twig\Error\RuntimeError;
 
 class UnwrapTwigExceptionListener
 {
-    private static array $exceptionsToUnwrap = [
-        NoContentResponseException::class,
-        RedirectResponseException::class,
-    ];
-
     /**
      * If an exception is encountered while rendering a Twig template, Twig
-     * will wrap the exception in a Twig\Error\RuntimeError. For cases that we
-     * want the exception to bubble, we need to unwrap the original exception
-     * again.
+     * will wrap the exception in a Twig\Error\RuntimeError. In case of our
+     * response exceptions, we need them to bubble, though. Therefore, we
+     * unwrap them again, here.
      */
     public function __invoke(ExceptionEvent $event): void
     {
         if (
             !($throwable = $event->getThrowable()) instanceof RuntimeError ||
-            ($previous = $throwable->getPrevious()) === null
+            !is_a($previous = $throwable->getPrevious(), ResponseException::class)
```

Comment:
```suggestion
            !($previous = $throwable->getPrevious()) instanceof ResponseException
```
As using `instanceof` in the first check and `is_a()` in the second looks confusing (to me ☺️)